eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -1836,6 +1836,13 @@
     <Bug pattern="UPM_UNCALLED_PRIVATE_METHOD"/>
   </Match>
 
+
+  <Match>
+    <Class name="com.azure.cosmos.implementation.routing.UInt128"/>
+    <Method name="compareTo"/>
+    <Bug pattern="SA_FIELD_SELF_COMPARISON"/>
```

Comment:
why do we need this ?

Reply:
I implements Comparable in Uint128, so implements compareTo, and got the complaining from spotbugs.